service/framework.go

```diff
@@ -256,8 +257,8 @@ func newCustomObjectFramework(config Config) (*framework.Framework, error) {
 				return nil, microerror.Mask(err)
 			}
 
-			meta.KeyVals["cluster"] = key.ClusterID(customObject)
-			meta.KeyVals["framework"] = "customobject"
+			meta.KeyVals["object"] = path.Join(customObject.Namespace, customObject.Name)
+			meta.KeyVals["groupVersionKind"] = customObject.GroupVersionKind().String()
```

Comment:
I removed cluster, but can restore. Let's discuss. I think we don't need that anymore. I'd like to move that stuff to operatorkit since it's pretty generic. That information is in `runtime.Object`.

Reply:
IMO cluster is super specific to the kvm-operator. I want to see which resources for which cluster is reconciled.

Reply:
Then `customObject.Name` is a cluster name. Isn't that sufficient? You can do `| grep Kind=Kvm | grep CLUSTER_ID` and all good. My motivation was to not clutter logs more.

Reply:
Ok cool. Works for me. Didn't had in mind the custom object's name is actually the cluster ID.